tests/integration/api_branch_test.go

```diff
@@ -139,6 +139,8 @@ func testAPICreateBranches(t *testing.T, giteaURL *url.URL) {
 			ExpectedHTTPStatus: http.StatusConflict,
 		},
 		// Trying to create from other branch (not default branch)
+		// ps: it can't test the case-sensitive behavior here, because on macOS, the branch can't be created by git, make the test fails quickly, before the database code
```

Comment:
```suggestion
		// ps: it can't test the case-sensitive behavior here: the "BRANCH_2" can't be created by git on a case-insensitive filesystem, it makes the test fail quickly before the database code.
```